build_docker.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+#
+# Runs `go build` with flags configured for docker distribution. All
+# it does differently from `go build` is burn git commit and version
+# information into the binaries inside docker, so that we can track down user
+# issues.
+
+set -eu
+
+eval $(./version/version.sh)
+
+GOFLAGS='xversion -ldflags '"-X tailscale.com/version.Long=${VERSION_LONG} -X tailscale.com/version.Short=${VERSION_SHORT} -X tailscale.com/version.GitCommit=${VERSION_GIT_HASH}"
+docker build --build-arg goflags_arg="'""${GOFLAGS}""'" -t tailscale:tailscale .
```

Comment:
can you add a newline at the end of this line, and also add a big fat warning banner to the top of this script like the Dockerfile itself has?